src/components/scene/inspector.js

```diff
@@ -0,0 +1,61 @@
+/* global AFRAME */
+var registerComponent = require('../../core/component').registerComponent;
+var pkg = require('../../../package');
+
+var INSPECTOR_URL = pkg.homepage + 'releases/' + pkg.version + '/aframe-inspector.min.js';
+
+module.exports.Component = registerComponent('inspector', {
+  schema: {
+    url: {default: INSPECTOR_URL}
+  },
+
+  init: function () {
+    this.onKeydown = this.onKeydown.bind(this);
+    window.addEventListener('keydown', this.onKeydown);
+  },
+
+  onKeydown: function (evt) {
+    // Alt + Ctrl + i
+    var shortcutPressed = evt.keyCode === 73 && evt.ctrlKey && evt.altKey;
+    var escape = evt.keyCode === 27;
```

Comment:
Yes. The escape shortcut is to close the editor that will be occupying the whole screen with the scene paused. The editor should be the only thing in the page listening for key presses.